src/api/common_api.js

```diff
@@ -848,5 +848,51 @@ module.exports = {
                 }
             }
         },
+        bucket_lifecycle_filter: {
+            type: 'object',
+            properties: {
+                and: {
+                    type: 'object',
```

Comment:
not sure what it this and? It seems like the filter needs to be an array of this properties. take a look here: https://docs.aws.amazon.com/AmazonS3/latest/userguide/lifecycle-configuration-examples.html

Reply:
As we talked, removed the `and`, and we will just do it when we parse the request.
also as this PR is not containing new/missing flows commented out the unused properties